src/Microsoft.ML.Data/Transforms/KeyToValueTransform.cs

```diff
@@ -32,7 +32,7 @@
 [assembly: LoadableClass(typeof(IRowMapper), typeof(KeyToValueTransform), null, typeof(SignatureLoadRowMapper),
     KeyToValueTransform.UserName, KeyToValueTransform.LoaderSignature)]
 
-namespace Microsoft.ML.Runtime.Data
+namespace Microsoft.ML.Transforms.CategoricalTransforms
```

Comment:
Would it be possible to align folders and namespaces (at least as much as possible)? Any class in the `Transforms.CategoricalTransforms` namespace should be in the `Transforms\CategoricalTransforms` folder. If it is important enough to separate the classes into namespaces, separating them into folders is equally important - so devs can find the classes.